### PR TITLE
feat: one bucket name

### PR DESCRIPTION
**Pull request by goshander**
Sometimes we have not permissions for listing all buckets with secret and access key

Need feature for specify one bucket and skip listing request

### CODE REVIEW

File: lib/storage.dart [hunk 1]
```diff
@@ -23,13 +24,14 @@ class Storage {
...
-    const folderName = "PilotS3";
-    final path = Directory("$documentPath\\$folderName");
-    if (!(await path.exists())) {
-      path.create();
+    final documentPath =
+        Platform.isLinux ? path.join(appDir.path, '.config') : appDir.path;
+    final folderName = Platform.isLinux ? "pilot-s3" : "PilotS3";
+    final configPath = Directory(path.join(documentPath, folderName));
+    if (!(await configPath.exists())) {
+      configPath.create(recursive: true);
```
Comment (AlexEbenrode) [inline lib/storage.dart:31]: May be remove path_provider then?

**Comment by AlexEbenrode** [comment]
I think need put "Listing all buckets" checkbox in settings page, if it's disabled then enable bucket textbox for specify bucket

File: lib/storage.dart (hunk 1 above)
Comment (goshander) [reply lib/storage.dart:31]: `path_provider` provide method  `getApplicationDocumentsDirectory` for home dir getting
`path` used just for join strings with os specific slash

both plugin needed i think

Comment (AlexEbenrode) [reply lib/storage.dart:31]: kk

**Review by AlexEbenrode — DISMISSED** [review]

**Review by AlexEbenrode — APPROVED** [review]

**Comment by goshander** [comment]
:tada: this issue has been resolved in version 1.0.0 :tada:

release is available on [GitHub](<github_release_url>)

**Pull request merged**